libs/pages/database/src/lib/feature/page-settings-feature/page-settings-feature.tsx

```diff
@@ -12,8 +13,11 @@ import PageSettings from '../../ui/page-settings/page-settings'
 
 export function PageSettingsFeature() {
   const { organizationId = '', projectId = '', environmentId = '', databaseId = '' } = useParams()
+  const { data: database } = useService({ serviceId: databaseId })
```

Comment:
`environmentId` is missing no ?

Reply:
It works without, but I'll add it thanks!

Reply:
https://github.com/Qovery/console/pull/1105/commits/cf936ba724107effac122efe7a92c661a743c6d5